src/md/bylaws.md

```diff
@@ -0,0 +1,381 @@
+#*** draft subject to top level project approval ***
+#Apache Apex Project Bylaws 
+
+This document defines the bylaws under which the Apache Apex project operates. It
+defines the roles and responsibilities of the project, who may vote, how voting works,
+how conflicts are resolved, etc.
+
+Apex is a project of the <a class="externalLink" href=
+"http://www.apache.org/foundation/">Apache Software Foundation</a>. The Foundation
+holds the copyright on Apache code including the code in the Apex codebase. The
+<a class="externalLink" href="http://www.apache.org/foundation/faq.html">Foundation
+FAQ</a> explains the operation and background of the foundation.
+
+Apex is typical of Apache projects in that it operates under a set of principles,
+known collectively as the "Apache Way". If you are new to Apache
+development, please refer to the <a class="externalLink" href=
+"http://incubator.apache.org/">Incubator project</a> for more information on how Apache
+projects operate.
+
+##Roles and Responsibilities
+
+Apache projects define a <a class="externalLink" href=
+"http://www.apache.org/foundation/how-it-works.html#roles">set of roles</a> with
+associated rights and responsibilities. These roles govern what tasks an individual
+may perform within the project. The roles are defined in the following sections:
+
+###Users
+
+The most important participants in the project are people who use our software.
+The majority of our developers start out as users and guide their development
+efforts from the user&rsquo;s perspective.
+
+Users contribute to the Apache projects by providing feedback to developers in
+the form of bug reports and feature suggestions. As well, users participate in the
+Apache community by helping other users on mailing lists and user support
+forums.
+
+###Contributors
+
+All of the volunteers who are contributing time, code, documentation, or
+resources to the Apex Project. A contributor that makes sustained, welcome
+contributions to the project may be invited to become a Committer, though the exact
+timing of such invitations depends on many factors.
+
+###Committers
+
+The project&rsquo;s Committers are responsible for the project&rsquo;s technical
+management. All committers have write access to the project&rsquo;s source
+repositories. Committers may cast binding votes on any technical discussion
+regarding the project.
+
+Committer access is by invitation only and must be approved by lazy consensus of
+the active PMC members. A Committer may request removal of their commit privileges
+by their own declaration. A committer will be considered
+&ldquo;emeritus/inactive&rdquo; by not contributing in any form to the project for
+over 1 year. An emeritus committer may request reinstatement of commit access from
```

Comment:
Just being picky here.. Can we make this "one year" instead of "1 year"... Just to make it consistent with what is there in "PMC" section.